src/main/java/paymelah/model/person/Address.java

```diff
@@ -31,7 +31,7 @@ public Address(String address) {
     }
 
     /**
-     * Returns true if a given string is a valid email.
+     * Returns true if a given string is a valid telegram handle.
```

Comment:
This is for the physical `Address` not email address. So should be "Returns true if a given string is a valid address"